### ISSUE TITLE
V4 Modal Dialog adds padding-right to body when scollbar is visible

### ISSUE THREAD

**Issue by oschn0r** [twbs/bootstrap#17635]
In latest 4.0alpha opening of a modal adds "padding-right: 1617px;" to the body. In my case in latest Chrome and Firefox but Safari is working fine. It only  happens when a visible scrollbar appears.

Example:
https://jsfiddle.net/cbdsn73r/1/

Edit: I give up with the verify bot... sorry folks.

**Comment by twbs-lmvtfy** [comment]
Hi @oschn0r!

You appear to have posted a live example (https://fiddle.jshell.net/cbdsn73r/1/show/light/), which is always a good first step. However, according to [the HTML5 validator](http://html5.validator.nu), **your example has some validation errors**, which might potentially be causing your issue:
* line 43, column 7 thru column 26: Start tag ```` body ```` seen but an element of the same type was already open.
* line 43, column 7 thru column 26: Cannot recover after last error. Any further errors will be ignored.

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by oschn0r** [comment]
Updated
https://jsfiddle.net/cbdsn73r/3/

**Comment by twbs-lmvtfy** [comment]
Hi @oschn0r!

You appear to have posted a live example (https://fiddle.jshell.net/cbdsn73r/3/show/light/), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* line 93, column 18: [E013](https://github.com/twbs/bootlint/wiki/E013): Only columns (`.col-*-*`) may be children of `.row`s
* line 94, column 18: [E013](https://github.com/twbs/bootlint/wiki/E013): Only columns (`.col-*-*`) may be children of `.row`s
* line 95, column 18: [E013](https://github.com/twbs/bootlint/wiki/E013): Only columns (`.col-*-*`) may be children of `.row`s
* line 96, column 18: [E013](https://github.com/twbs/bootlint/wiki/E013): Only columns (`.col-*-*`) may be children of `.row`s
* line 97, column 18: [E013](https://github.com/twbs/bootlint/wiki/E013): Only columns (`.col-*-*`) may be children of `.row`s
* line 98, column 18: [E013](https://github.com/twbs/bootlint/wiki/E013): Only columns (`.col-*-*`) may be children of `.row`s
* line 99, column 18: [E013](https://github.com/twbs/bootlint/wiki/E013): Only columns (`.col-*-*`) may be children of `.row`s
* line 100, column 18: [E013](https://github.com/twbs/bootlint/wiki/E013): Only columns (`.col-*-*`) may be children of `.row`s
* line 101, column 18: [E013](https://github.com/twbs/bootlint/wiki/E013): Only columns (`.col-*-*`) may be children of `.row`s
* line 102, column 18: [E013](https://github.com/twbs/bootlint/wiki/E013): Only columns (`.col-*-*`) may be children of `.row`s
* line 103, column 18: [E013](https://github.com/twbs/bootlint/wiki/E013): Only columns (`.col-*-*`) may be children of `.row`s
* line 104, column 18: [E013](https://github.com/twbs/bootlint/wiki/E013): Only columns (`.col-*-*`) may be children of `.row`s
* line 105, column 18: [E013](https://github.com/twbs/bootlint/wiki/E013): Only columns (`.col-*-*`) may be children of `.row`s
* line 106, column 18: [E013](https://github.com/twbs/bootlint/wiki/E013): Only columns (`.col-*-*`) may be children of `.row`s
* line 107, column 18: [E013](https://github.com/twbs/bootlint/wiki/E013): Only columns (`.col-*-*`) may be children of `.row`s
* line 108, column 18: [E013](https://github.com/twbs/bootlint/wiki/E013): Only columns (`.col-*-*`) may be children of `.row`s
* line 109, column 18: [E013](https://github.com/twbs/bootlint/wiki/E013): Only columns (`.col-*-*`) may be children of `.row`s
* line 110, column 18: [E013](https://github.com/twbs/bootlint/wiki/E013): Only columns (`.col-*-*`) may be children of `.row`s
* line 111, column 18: [E013](https://github.com/twbs/bootlint/wiki/E013): Only columns (`.col-*-*`) may be children of `.row`s
* line 112, column 18: [E013](https://github.com/twbs/bootlint/wiki/E013): Only columns (`.col-*-*`) may be children of `.row`s
* line 113, column 18: [E013](https://github.com/twbs/bootlint/wiki/E013): Only columns (`.col-*-*`) may be children of `.row`s
* line 114, column 18: [E013](https://github.com/twbs/bootlint/wiki/E013): Only columns (`.col-*-*`) may be children of `.row`s
* line 115, column 18: [E013](https://github.com/twbs/bootlint/wiki/E013): Only columns (`.col-*-*`) may be children of `.row`s
* line 116, column 18: [E013](https://github.com/twbs/bootlint/wiki/E013): Only columns (`.col-*-*`) may be children of `.row`s
* line 117, column 18: [E013](https://github.com/twbs/bootlint/wiki/E013): Only columns (`.col-*-*`) may be children of `.row`s
* line 118, column 18: [E013](https://github.com/twbs/bootlint/wiki/E013): Only columns (`.col-*-*`) may be children of `.row`s
* line 119, column 18: [E013](https://github.com/twbs/bootlint/wiki/E013): Only columns (`.col-*-*`) may be children of `.row`s
* line 120, column 18: [E013](https://github.com/twbs/bootlint/wiki/E013): Only columns (`.col-*-*`) may be children of `.row`s
* line 121, column 18: [E013](https://github.com/twbs/bootlint/wiki/E013): Only columns (`.col-*-*`) may be children of `.row`s
* line 122, column 18: [E013](https://github.com/twbs/bootlint/wiki/E013): Only columns (`.col-*-*`) may be children of `.row`s
* line 123, column 18: [E013](https://github.com/twbs/bootlint/wiki/E013): Only columns (`.col-*-*`) may be children of `.row`s
* line 124, column 18: [E013](https://github.com/twbs/bootlint/wiki/E013): Only columns (`.col-*-*`) may be children of `.row`s
* line 125, column 18: [E013](https://github.com/twbs/bootlint/wiki/E013): Only columns (`.col-*-*`) may be children of `.row`s
* line 126, column 18: [E013](https://github.com/twbs/bootlint/wiki/E013): Only columns (`.col-*-*`) may be children of `.row`s
* line 127, column 18: [E013](https://github.com/twbs/bootlint/wiki/E013): Only columns (`.col-*-*`) may be children of `.row`s
* line 128, column 18: [E013](https://github.com/twbs/bootlint/wiki/E013): Only columns (`.col-*-*`) may be children of `.row`s
* line 129, column 18: [E013](https://github.com/twbs/bootlint/wiki/E013): Only columns (`.col-*-*`) may be children of `.row`s
* line 130, column 18: [E013](https://github.com/twbs/bootlint/wiki/E013): Only columns (`.col-*-*`) may be children of `.row`s
* line 131, column 18: [E013](https://github.com/twbs/bootlint/wiki/E013): Only columns (`.col-*-*`) may be children of `.row`s
* line 60, column 9: [W012](https://github.com/twbs/bootlint/wiki/W012): `.container` or `.container-fluid` should be the first child inside of a `.navbar`

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by cvrebert** [comment]
Duplicate of #17399.
